tests/test_db_runtime_security.py

```
@@ -576,14 +596,16 @@ async def run_test() -> tuple[str, str]:
 
             close_started = threading.Event()
             close_done = threading.Event()
-            close_error: list[Exception] = []
+            close_error: Future[None] = Future()
 
             def _close_manager() -> None:
                 close_started.set()
                 try:
                     self.manager.close()
                 except Exception as err:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)